src/ImageSharp/Processing/ResizeMode.cs

```diff
@@ -42,6 +42,11 @@ public enum ResizeMode
         /// <summary>
         /// Stretches the resized image to fit the bounds of its container.
         /// </summary>
-        Stretch
+        Stretch,
+
+        /// <summary>
+        /// The target location and size of the resized image has been manually set.
+        /// </summary>
+        Manual
```

Comment:
Wow, nice solution! 👍
(I was stuck to find a clean way to manage this.)

Reply:
Yeah, surprised myself there!